apps/admin-api/src/schemas/override.schema.ts

```diff
@@ -0,0 +1,33 @@
+import * as mongoose from 'mongoose'
+
+import { GenEdType, StudyProgram } from '@admin-api/graphql'
+
+export const GenEdSchema = new mongoose.Schema({
+  genEdType: {
+    type: String,
+    required: true,
+    enum: Object.values(GenEdType),
+  },
+  sections: { type: [String], required: true },
+})
+
+export const OverrideSchema = new mongoose.Schema({
+  courseNo: { type: String, required: true },
+  studyProgram: { type: String, required: true, enum: Object.values(StudyProgram) },
+  semester: { type: String, required: true },
+  academicYear: { type: String, required: true },
+  genEd: { type: GenEdSchema },
```

Comment:
don't forget that we'll remove sections, study program, semester, and academic year from gened, leaving only courseNo as only unique identifier. just FYI no need to do it in this PR